src/connector/src/schema/schema_registry/client.rs

```diff
@@ -135,12 +138,25 @@ impl Client {
             client: self.inner.clone(),
             path: path.iter().map(|p| p.to_string()).collect_vec(),
         });
+
+        let retry_strategy = ExponentialBackoff::from_millis(100)
+            .factor(2)
+            .max_delay(Duration::from_secs(1))
+            .take(3);
```

Comment:
Let's add a jitter here as well because we want to reduce the peak QPS to schema registry. Example: https://github.com/risingwavelabs/risingwave/blob/60ebe13ccf40cb69ced1849c2ca28ff3c393a909/src/stream/src/executor/source/mod.rs#L172

I am also thinking whether we should add a few ms delay with some randomness for the initial request before the retry kicks in to reduce the QPS for the 1st attempt.